src/caret_analyze/infra/lttng/lttng.py

```diff
@@ -90,10 +90,46 @@ def _cache_path(self, events_path: str) -> str:
 
     @staticmethod
     def _trace_dir_exists(path: str) -> bool:
+        """
+        Check whether trace dir exists.
+
+        Parameters
+        ----------
+        path : str
+            path to trace dir.
```

Comment:
`path` -> `Path`

Reply:
addressed in :[e1ba6f1](https://github.com/tier4/CARET_analyze/pull/189/commits/e1ba6f106edfd31e8e5f83e8f4e4064a9cdfd021)